2.4/contrib/common.sh

```diff
@@ -160,10 +161,10 @@ function mongo_create_users() {
 # setup_keyfile fixes the bug in mounting the Kubernetes 'Secret' volume that
 # mounts the secret files with 'too open' permissions.
 function setup_keyfile() {
-  if [ ! -f "${MONGODB_KEYFILE_SOURCE_PATH}" ]; then
-    echo "ERROR: You have to provide the 'keyfile' in ${MONGODB_KEYFILE_SOURCE_PATH}"
+  if [ -z "${MONGODB_KEYFILE_VALUE}" ]; then
```

Comment:
you can't just add mandatory new env variables like this.  our existing DB templates don't provide them and the doc needs to be updated if you're going to do something like this.